index.js

```diff
@@ -22,4 +235,25 @@ document.querySelectorAll('.close').forEach((n) => n
   .addEventListener('click', () => {
     hmgIcon.classList.remove('active');
     overlayMenu.classList.remove('active');
-  }));
\ No newline at end of file
+  }));
+
+// Contact Form Validation
+
+const submitBtn = document.getElementById('submit-btn');
+const error = document.getElementById('error-msg');
+const emailInput = document.getElementById('email-at');
+
+error.style.color = 'red';
+error.style.padding = '0.5em';
+error.style.textAlign = 'center';
+
+submitBtn.onclick = (event) => {
+  const emailValue = document.getElementById('email-at').value;
+  const lowerEmail = emailValue.toLowerCase();
+
+  if (emailValue !== lowerEmail) {
+    error.textContent = 'Please check the email format';
+    emailInput.classList.toggle('email-error-msg');
+    event.preventDefault();
+  }
+};
```

Comment:
- [ ] OPTIONAL: RegEx allows us to check for patterns in text strings such as trying to match a valid email address or password. One of the great superpowers of RegEx is being able to define your own search criteria for a pattern to fit your needs, and it is like a language of its own. For these reasons, I would suggest you also implement some regex to check the `email input` value. eg: `const emailRegex = /^[a-z0-9@.]+$/;` and the compare it with the input value. Try it and you'll the magic of regex in javascript 😄 😊